packages/foundry/loadTemplate.ts

```diff
@@ -0,0 +1,10 @@
+import { Configuration } from "./cli";
+import { ensureDir } from "fs-extra";
+import degit from "degit";
+
+export const LoadTemplate = async (config: Configuration): Promise<void> => {
```

Comment:
I wonder why `LoadTemplate` is in an standalone file while `ReplaceInFile` is inlined in the `index` file? Is there a difference between both functions that led you to this decision?

Reply:
I think a more appropriate name for the function would be something like `createProject` or `generateProject`.